src/App.vue

```diff
@@ -183,50 +183,17 @@ import * as Dialogs from '@/dialogs'
 import * as Views from '@/views'
 
 // Mixins, interfaces, etc
-import {
-  CommonMixin,
-  DateMixin,
-  FilingTemplateMixin,
-  NameRequestMixin
-} from '@/mixins'
-import {
-  AccountInformationIF,
-  ActionBindingIF,
-  AddressIF,
-  BreadcrumbIF,
-  BusinessIF,
-  CompletingPartyIF,
-  ConfirmDialogType,
-  EmptyFees,
-  FilingDataIF,
-  OrgInformationIF,
-  ResourceIF,
-  StepIF
-} from '@/interfaces'
-import {
-  DissolutionResources,
-  IncorporationResources,
-  RegistrationResources,
-  RestorationResources,
-  getEntityDashboardBreadcrumb,
-  getMyBusinessRegistryBreadcrumb,
-  getRegistryDashboardBreadcrumb,
-  getSbcStaffDashboardBreadcrumb,
-  getStaffDashboardBreadcrumb
-} from '@/resources'
+import { CommonMixin, DateMixin, FilingTemplateMixin, NameRequestMixin } from '@/mixins'
+import { AccountInformationIF, ActionBindingIF, AddressIF, BreadcrumbIF, BusinessIF, CompletingPartyIF,
+  ConfirmDialogType, EmptyFees, FilingDataIF, OrgInformationIF, ResourceIF, StepIF } from '@/interfaces'
+import { DissolutionResources, IncorporationResources, RegistrationResources, RestorationResources,
+  getEntityDashboardBreadcrumb, getMyBusinessRegistryBreadcrumb, getRegistryDashboardBreadcrumb,
+  getSbcStaffDashboardBreadcrumb, getStaffDashboardBreadcrumb } from '@/resources'
```

Comment:
I prefer all-on-one-line (with line breaks) instead of the old, tall import.

What do you guys think?

Reply:
I prefer all-on-one-line as well.